Bottom: ee439139fdbda633ae1d68dd6e1e1190025244e1
Top:    d309b091ae30a588b53264797770609c97c799d7
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-24 16:24:41 +0800

Refresh of autofs-5.0.7-use-compiler-determined-by-configure-instead-of-hard-coded-ones.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 72d9135..ebabf21 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -61,6 +61,7 @@
 - fix hesiod check error and use correct $(LIBS) setting.
 - fix dead LDAP symbolic link when LDAP support is disabled.
 - add missing libtirpc lib to mount_nfs.so when TIRPC enabled.
+- use compiler determined by configure instead of hard-coded ones.
 
 25/07/2012 autofs-5.0.7
 =======================
